app/templates/partials/questions/general.html

```diff
@@ -18,7 +18,12 @@
 
         <legend class="question__legend">
           {%- if question_title -%}
-            <span class="question__title neptune">{{question_title}}</span>
+            <span class="question__title neptune">
+                {%- if question_number -%}
+                    <span>{{question_number}}. </span>
+                {% endif %}
+                {{question_title}}
+            </span>
```

Comment:
There are `question_titles` and `question_guidance` jinja variables that we should use if possible